nimbus-features/messagingFeature.yaml

```diff
@@ -134,16 +134,6 @@ features:
               text: ResearchSurface/Body.Text.v112
               button-label: ResearchSurface/PrimaryButton.Label.v112
               action: https://www.macrumors.com
-            engagement-notification:
```

Comment:
This change is needed for UI tests changes? Or for another reason?

Reply:
It was removed because we get this data from Nimbus. It was just there to quickly add it for testing but I don't think it's needed anymore. It was not executed anymore anyway (hence the trigger `- NEVER`).